src/Services/AiUsageSummaryService.php

```
@@ -0,0 +1,245 @@
+        return $feature !== '' ? $feature : 'unspecified';
+    }
+
+    private function friendlyFeatureName(string $feature): string
+    {
+        return ucfirst(str_replace('_', ' ', $feature));
+    }
+
+    private function createdAt(string $createdAt): \DateTimeImmutable
+    {
+        try {
+            return new \DateTimeImmutable($createdAt !== '' ? $createdAt : 'now');
```

PHPMD (2.15.0)

[error] Missing class import via use statement (line '159', column '24'). (undefined)

(MissingImport)